### DISCUSSION TITLE
Input data standards planning thread

### DISCUSSION THREAD

**Discussion by sbordyak** [CIRDLES/Tripoli#7] [Ideas]
Use this thread to discuss desired input types and naming standards for input variables.  

Suggestions and requests can be collated into the google doc here: [Tripoli input data standards](https://docs.google.com/document/d/1Ox-mMCckNRAqapxTBlkumetChmcTHF3PoH85K3F08FQ/edit?usp=sharing)

**Comment by sbordyak** [comment]
To begin, here are the variables included in the output file created by Isotopx.  These data are used as the input for the initial development of the MC algorithm.

Header Information
> Version															
> Filename															
> Sample No														
> Sample ID													
> Sample Type															
> Analysis name															
> User name															
> Method name														
> MS Tune file															
> Inlet file															
> Serial Number															
> Centre Channel														
> DataDump Level														
> Acquire Date														
> Cycles per Block															
> Num. Blocks															
> Num. Sequences	

Time Sampled Data
> Cycle ID
> Block
> Cycle
> Integ. (Time Sample?)
> Time
> Mass
> Voltage/count for each collector

**Comment by bowring** [comment]
I have extracted the table schema from Isotopx SQLlite TIMSAD file into the [Tripoli input data standards](https://docs.google.com/document/d/1Ox-mMCckNRAqapxTBlkumetChmcTHF3PoH85K3F08FQ/edit?usp=sharing) .  I can find no data dictionary that defines the fields.  Does anyone know where to find one?
